docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+version: '3'
+services:
+  jupiter:
+    container_name: "cdapython-notebook"
+    image: jupyter/minimal-notebook:latest
+    working_dir: /home/joyvan/work
```

Comment:
'joyvan?'   This might be something that we want to maybe update in the future.  This seems a little sketch.